convert_fendl.py

```diff
@@ -41,90 +42,308 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
                     default='earliest', help="Output HDF5 versioning. Use "
                     "'earliest' for backwards compatibility or 'latest' for "
                     "performance")
-parser.add_argument('-r', '--release', choices=['3.1a', '3.1d'],
+parser.add_argument('-r', '--release', choices=['3.1a', '3.1d', '3.0', '2.1'],
```

Comment:
rearrange this so they are in order of release newest to oldest
'3.1d', '3.1a', '3.0', '2.1'

Reply:
Could the description text on line 15-17 also be updated so it includes all the versions

Reply:
Thanks